.github/workflows/test-cron.yaml

```diff
@@ -253,18 +246,34 @@ jobs:
       run: './pants --version
 
         '
+    - name: Run smoke tests
```

Comment:
No harm in running smoke tests as part of bootstrapping, as they should be really fast, and it saved me having to make this conditional. But we can do so if it turns out to be a problem.